i6_models/parts/conformer/convolution.py

```diff
@@ -0,0 +1,59 @@
+from torch import nn
+
+
+class ConformerConvolutionV1(nn.Module):
```

Comment:
In RETURNN-common, you called this `ConformerConvBlock`, not `ConformerConvolution`. Why this name change?